zadaci/sa-kolokvijuma/2020/RA/T5/G3/zadatak.rst

```diff
@@ -0,0 +1,74 @@
+
+
+Iz zadate ulazne datoteke ``artikli.txt`` učitati podatke u jednostruko
+spregnutu listu, gde struktura ``artikal_st`` sadrži sledeća polja:
+
+- serijski_broj (ceo broj),
+
+- naziv (jedna reč, maksimalno 13 karaktera),
+
+- kolicina (pozitivan ceo broj),
+
+- jedinicna_cena (realan broj) i
+
+- ukupna_cena (realan broj).
+
+Struktura ``artikal_st``, takođe sadrži i polja potrebna za pravilno
+formiranje jednostruko spregnute liste. Prilikom formiranja jednostruko
+spregnute liste voditi računa o tome da je potrebno elemente sortirati u
+rastućem redosledu serijskog broja. Polje strukture ``ukupna_cena``
+izračunati na sledeći način: ``ukupna_cena = kolicina * jedinicna_cena``.
+
+Potrebno je kupiti sve one artikle iz ponude prodavnice čiji zbir
+ukupnih cena ne premašuje vrednost zadatog budžeta ``budzet`` . Nakon
+uspešne kupovine, ispisati sve informacije o svim kupljenim artiklima.
+Uz to, ispisati još i iznos preostalog budžeta i informaciju o broju
+kupljenih artikala. Ako se ispostavi da je budžet nedovoljan za kupovinu
+bilo kog artikla iz ponude, ispisati odgovarajuću poruku i izaći iz
+programa sa izlaznim kodom 333.
+
+Ako se za vrednost argumenta komandne linije ``budzet`` unese negativna
+vrednost ispisati poruku o grešci i iz programa izaći sa izlaznim kodom
+22.
+
+Primer poziva::
+
+    ./a.out 22800 artikli.txt
+
+sa zadatim argumentom ``budzet = 22800`` i zadatim ulazom u
+datoteci ``artikli.txt``::
+
+    123.19 3 limun 2396
+    4399.99 25 trenerka 1179
+    109.11 8 tecni_sapun 7421
+    45.01 5 luk 7965
+    999.99 3 duks 4598
+    89.99 22 banane 8942
+    145.11 78 pasta_za_zube 1124
+    189.22 23 arf 3211
+    5899.39 7 pantalone 1698
+    215.49 13 ananas 1147
+    386.29 49 regenerator 3352
+    2354.99 15 farmerke 4452
```

Comment:
Again, please include input file you provided to escape copy/paste errors. Remove one colon in line 39 and replace lines 41-52 with the following:

``.. literalinclude:: artikli.txt``